packages/example-app-base/src/styles/_mixins.scss

```diff
@@ -34,3 +34,11 @@
     pointer-events: none;
   }
 }
+
+// High contrast mode mixins
```

Comment:
Could you add a reference to `~office-ui-fabric-react/lib/common/_highContrast.scss` instead? (that may not be quite the right path formatting)

Reply:
That doesn't seem to work. Looks like `lib/common` has an already-javascript version, so can't be imported by the sass loader.

Reply:
Ah it's dist. Similar example: `@import '~office-ui-fabric-core/dist/sass/References';`

Reply:
Wait sorry that's an import from core...

Reply:
Interesting, for OUFR imports I see other places using both src and dist. So...pick one?
```
@import '~office-ui-fabric-react/src/common/common';
@import '~office-ui-fabric-react/dist/sass/References';
```